package/src/components/ConnectButton/ConnectButton.tsx

```diff
@@ -40,52 +44,81 @@ export const ConnectButton = () => {
     connect: connectWalletConnect,
   } = useWalletConnectWallet({ onClose: closeModal });
 
+  const { ensName } = useEnsName({
+    address: address?.[0] ? address[0] : '0x0',
+  });
```

Comment:
could use `address?.[0] ?? '0x0'` instead of `address?.[0] ? address[0] : '0x0'`